### PR TITLE
Testing/attendes

### PR DESCRIPTION
**Pull request by RodrigoACG**
## Type of Change
- [ ] New Feature
- [x] Debugging
- [x] Refactor
## Describe Changes Made
Testing for the attendees and now 100% passing in simple cov. Also refactored a little bit of the index action

## PR Checklist
- [x] Added Reviewer
- [x] Followed TDD, And Test are Passing

### CODE REVIEW

File: app/controllers/api/v1/attendees_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def destroy
   end
 
   def index
-    attendees = Attendee.where(event_id: params[:event_id])
+    attendees = Attendee.all
```
Comment (GBowman1) [inline app/controllers/api/v1/attendees_controller.rb]: this might create a conflict with how the attendees for an event is being accessed

**Comment by GBowman1** [comment]
image

**Comment by GBowman1** [comment]
we are trying to get an index of a specific event so we were using the event_id param to search for it

**Comment by RodrigoACG** [comment]
> we are trying to get an index of a specific event so we were using the event_id param to search for it

ohh okay ill work on that now and try to fix that

**Comment by RodrigoACG** [comment]
> > we are trying to get an index of a specific event so we were using the event_id param to search for it
> 
> ohh okay ill work on that now and try to fix that

It is now fixed

**Review by GBowman1 — APPROVED** [review]

**Pull request merged**